test/performance/odp_l2fwd.c

```diff
@@ -1507,10 +1512,12 @@ static void usage(char *progname)
 	       "  -x, --vec_size <num>    Vector size (default %i).\n"
 	       "  -z, --vec_tmo_ns <ns>   Vector timeout in ns (default %llu ns).\n"
 	       "  -P, --promisc_mode      Enable promiscuous mode.\n"
+	       "  -l, --packet_len <len>  Maximum length of packets supported(default %d).\n"
+	       "  -L, --seg_len <len>     Packet pool segment length(default %d).\n"
```

Comment:
One more nitpick here: space before "(default", just like the other defaults, above.

Reply:
ACK